registry/app.py

```diff
@@ -57,6 +58,18 @@ def translate(input, rodne_cislo):
         else:
             return input
 
+    @app.template_filter("capitalize")
+    def capitalize(input):
+        output = None
+        for match in re.finditer(r"\w+", input):
+            start, end = match.span()
+            word = input[start:end]
+            if word.isupper():
+                if output is None:
+                    output = list(input)
```

Comment:
Nemůžem tohle udělat rovnou? Způsobíme tím, že se to občas udělá zbytečně, ale zase se tím ušetří dvě podmínky a zčitelní kód.

```suggestion
```

Reply:
Nejsem úplně nadšený z toho, že používáme seznam pro ukládání řetězců jenom proto, že v něm můžeme čitelněji nahrazovat. Není na to ve standardní knihovně nějaká funkce? A je vlastně `output = input[0:start] + word.capitalize() + input[end:]` tak nečitelné?